src/domain/event/__tests__/EventPage.test.tsx

```diff
@@ -527,6 +529,31 @@ it('opens expanded area with enrolment button when clicked', async () => {
   });
 });
 
+it('expanded area does not have an enrolment button if enrollment has not yet begun', async () => {
+  advanceTo(new Date(2020, 6, 12)); // Before enrolmentStart (2020-07-13T06:00:00+00:00)
+  renderComponent();
+  await waitForRequestsToComplete();
+
+  const occurrenceRow = within(screen.getAllByRole('row')[8]);
+
+  userEvent.click(
+    occurrenceRow.getByRole('button', {
+      name: occurrenceMessages.buttonShowDetails,
+    })
+  );
+
+  await waitFor(() => {
+    expect(screen.queryByText(data.venueDescription)).toBeInTheDocument();
+  });
+
+  expect(
+    screen.queryByRole('button', {
+      name: 'Ilmoittaudu',
+    })
+  ).toEqual(null);
```

Comment:
`).not.toBeInTheDocument();` would be consistent with how these checks are done in other tests

Reply:
`).not.toBeInTheDocument();` might even give better error message in console if this assertion fails

Reply:
I had this in the first version, but refactored it later. Seems more precise to check for `null` because it's a value that doesn't really relate with the DOM. But I can change this.

Reply:
Of course this would maybe be the most readable way to do this. But I understand your point of view
```
const enrolmentButton = screen.queryByRole('button', {
      name: 'Ilmoittaudu',
 });
 expect(enrolmentButton).not.toBeInTheDocument();
```

Reply:
But I think most importantly `toBeInTheDocument` should be used to stay consistent in this project